proposals/pr-contrib-guide.md

```diff
@@ -0,0 +1,471 @@
+   appendix](#changes-not-available-for-contribution) for details.
+1. Feature requests must have at least 10 :+1: reactions.
+1. The maintainers agree with the implementation plan, including any design
+   discussions (e.g., public interface design, edge cases, backwards
+   compatibility concerns).
+
+If the criteria are not satisfied, the maintainers should explicitly comment
+what criteria are not satsified and how the community can satisfy them. All
+discussions must be resolved prior to making a contribution so that feedback
+during the review process focuses on the implementation of the agreed upon
+design.
+
+Issues that do not receive enough :+1: reactions and do not have any activty for
+a year are considered stale. Without further interaction, they will be closed.
+
+### Contribution-ready stage
+
+A GitHub issue enters the contribution ready stage when a maintainer publicly
+labels the issue ready for contribution. This stage indicates any user is
+welcome to initiate a contribution via a GitHub pull request. All contribution
+ready issues are listed publicly for the community to discover.
+
+### Implementation stage
+
+A GitHub issue enters the implementation stage once a user submits a GitHub pull
+request implementing the change. This stage indicates that a contribution has
+been submitted for the issue, but the maintainers have not committed to a full
+review of the contribution. The maintainers will respond to let the contributor
+know to expect a preliminary review within 5 business days to confirm:
+
+1. It is implemented in the manner described in the issue.
+1. It meets the criteria specified in the contribution guide. See the
+   [appendix](#preliminary-review-criteria) for details.
+
+If the criteria are not satisfied, the maintainers will comment on the issue
+specifically indicating what work is still required. If the contributor has
+questions, they should refer to the contributing guide documentation or ask the
+maintainers to clarify. Once the contributor makes the required changes, the
+maintainers will follow up within 5 business days.
+
+If the contributor is no longer actively engaged by responding to comments from
+maintainers or making code changes for more than 30 business days, the
+contribution is stale. If not, the pull request will be closed and the
+corresponding issue will be marked as ready for contribution.
+
+### Ready for review stage
+
+Once the preliminary review criteria from the implementation stage have been
+met, the tracking GitHub issue is labeled as ready for review and added to a
+queue for maintainer review. The maintainers will select issues for full review
+from this queue. The issues will be prioritized for review publicly based on the
+following criteria in descending order of importance:
+
+1. Bugs, documentation changes, and feature requests will be prioritized in that
+   order.
+1. The time available to the maintainers will be considered against the
```

Comment:
That seems like a design discussion. It would probably take a review to determine many of these things - not sure they would be identified in a recurring planning and prioritization meeting.

Reply:
Going to reword this to keep in the discretion of the maintainers for pulling stuff up, but in general it will be a FIFO queue with some insertion sort for new issues. Old issues/reviews shouldn't be moved up, but new ones might come up.

Reply:
added to rationale some discussion for this.